internal/formats/cyclonedx13xml/test-fixtures/snapshot/TestCycloneDxDirectoryPresenter.golden

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<bom xmlns="http://cyclonedx.org/schema/bom/1.2" version="1" serialNumber="urn:uuid:5404937f-72d6-44a2-8e9b-954305ecb4f6">
+<bom xmlns="http://cyclonedx.org/schema/bom/1.3" serialNumber="371e31c0-d99c-4816-986e-5485246b47e3" version="1">
```

Comment:
Is this new serial number correct for 1.3?
https://cyclonedx.org/docs/1.3/json/#serialNumber

Reply:
I was not right, the integration test also complained